src/main/resources/org/jenkinsci/plugins/GithubSecurityRealm/config.jelly

```diff
@@ -16,5 +16,8 @@
 		<f:entry title="Client Secret" field="clientSecret" help="/plugin/github-oauth/help/realm/client-secret-help.html">
 			<f:textbox />
 		</f:entry>
+		<f:entry title="OAuth Scope(s)" field="oauthScopes" help="/plugin/github-oauth/help/realm/oauth-scopes-help.html">
+			<f:textbox default="repo,read:org" />
```

Comment:
I disagree. My first instinct reading this was "Yes, of course, checkboxes are better!"
However, after a little bit of thought, I think the advantages are illusory.

This isn't an entry field for an end user, it's for the admin.
Are we really worried that the admin might screw up the install by editing this?
The GitHub API URI is already exposed, along with a million and one other ways to bork the server.

The text box exposes to me a way of interacting directly with GitHub OAuth.
If I use other plugins which depend on this one, and they require unusual scopes, I am free to edit this field.
Not being a java developer, without this being a modifiable field I'd end up filing a duplicate of JENKINS-27691

That is to say, remember that this solves both
https://issues.jenkins-ci.org/browse/JENKINS-23324
and
https://issues.jenkins-ci.org/browse/JENKINS-27691
Not just the former.

Now, checkboxes + a text entry field would be fine, but why bother?